src/priv-csrs.tex

```diff
@@ -375,6 +378,9 @@ \section{CSR Listing}
 \tt 0x7A1 & MRW &\tt tdata1 & First Debug/Trace trigger data register. \\
 \tt 0x7A2 & MRW &\tt tdata2 & Second Debug/Trace trigger data register. \\
 \tt 0x7A3 & MRW &\tt tdata3 & Third Debug/Trace trigger data register. \\
+\tt 0x5A8 & SRW &\tt scontext & Supervisor-mode context register. \\
+\tt 0x6A8 & HRW &\tt hcontext & Hypervisor-mode context register. \\
```

Comment:
The `scontext` and `hcontext` CSRs need to go into the respective tables for supervisor CSRs and hypervisor CSRs.

Reply:
done